File: src/SPAAngular/scp-spa-angular/src/app/models/pagedList/PagedListPessoaFisica.ts
import { Guid } from 'guid-typescript';
import { PessoaFisicaFilter } from '../pessoa-fisica/pessoa-fisica-filter';
import { PagedList } from './pagedList';

export class PagedListPessoaFisica extends PagedList {

    pessoaFisicaFilter: PessoaFisicaFilter;

}

File: src/CrossCutting/SCP.CrossCutting/MapModels/PagedList/PagedListPessoaFisica/PagedListPessoaFisica.cs
using SCP.CrossCutting.MapModels.PessoaFisica;

namespace SCP.CrossCutting.MapModels.PagedList.PagedListPessoaFisica
{
    public class PagedListPessoaFisica : PagedList
    {
        public PessoaFisicaFilter PessoaFisicaFilter { get; set; }
    }
}
